jcl/src/jdk.management/share/classes/com/ibm/lang/management/internal/ExtendedOperatingSystemMXBeanImpl.java

```diff
@@ -154,18 +154,15 @@ private double calculateProcessCpuLoad(long endTs, long endCpuTime, long startTs
 	 */
 	@Override
 	public final long getFreePhysicalMemorySize() {
-		return this.getFreePhysicalMemorySizeImpl();
+		try {
+			memObj = retrieveMemoryUsage(memObj);
```

Comment:
Does this affect performance? How much longer does it take to call retrieveMemoryUsage() than j9vmem_get_available_physical_memory().